### PR TITLE
USAGOV-1912: sub page breadcrumb spacing update

### PR DESCRIPTION
**Pull request by developer43**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1912

## Description

Update spacing above and below the breadcrumb on sub pages
## Type of Changes

- [ ] New Feature
- [ ] Bugfix
- [x] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [ ] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [ ] Requires New Config
- [ ] Requires New Content

### Validation Steps
This is a visual test to make sure the spacing above and below the breadcrumb looks the same on desktop, tablet and mobile.
Goto http://localhost/facts-figures on desktop. Look at the spacing above and below the breadcrumb and check the page in tablet and mobile to see if the spacing looks the same.

Screenshot 2024-09-04 at 7 23 56 AM

Screenshot 2024-09-04 at 7 24 30 AM

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss [hunk 1]
```diff
@@ -73,8 +73,7 @@ i.e.
 @media (min-width: 40em) {
   .usa-section {
       padding-bottom: 4rem;
-      padding-top: 3rem;
-  }
```
Comment (scottqueen-bixal) [inline web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss:77]: this missing closing bracket is causing an error while compiling the theme


```
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
[17:16:54] Finished 'buildSprite' after 177 ms
[17:16:54] Starting 'renameSprite'...
[17:16:54] Finished 'renameSprite' after 2.4 ms
[17:16:54] Starting 'cleanSprite'...
[17:16:54] Finished 'cleanSprite' after 3.64 ms
sass/_uswds-theme-custom-styles.scss
Error: expected "}".
    ╷
955 │ }
    │  ^
    ╵
  sass/_uswds-theme-custom-styles.scss 955:2  @forward
  sass/styles.scss 72:1                       root stylesheet
[17:16:58] Finished 'buildSass' after 3.9 s
[17:16:58] Finished 'update' after 4.99 s
```

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

File: web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss (hunk 1 above)
Comment (developer43) [reply web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss:77]: @scottqueen-bixal thanks Scott, the issue is resolved and ready to approve.

Comment (scottqueen-bixal) [reply web/themes/custom/usagov/sass/_uswds-theme-custom-styles.scss:77]: this is now compiling

**Comment by scottqueen-bixal** [comment]
confirming that the theme compiles as expected, and our successful changes from https://github.com/usagov/usagov-2021/pull/1917 are not impacted on the `benefit-finder/* paths`

**Review by scottqueen-bixal — DISMISSED** [review]

**Comment by scottqueen-bixal** [comment]
Tested by following these steps:

- [x] pull changes locally
- [x] rebuild theme
- [x] clear cache
- [x] navigate to `/benefit-finder/death`
- [x] inspect element `#block-usagov-breadcrumbs`
- [x] expect `height: 4rem;`

Screenshot 2024-09-05 at 10 11 40 AM

- [x] inspect child element  of `class="bf-qual-survey grid-container"` 
- [x] expect that the `4rem` of height initially assigned (see prod) is no longer assigned

Screenshot 2024-09-05 at 10 13 33 AM

@developer43  Since qualtrics doesn't render the form on my local I won't be able to validate with a screen capture that the form is rendering as expected - once on dev we can poke around and validate again.

- [x] navigate to `/benefit-finder`
- [x] inspect element with class, `grid-row breadcrumb-spacing` 
- [x] expect `height: 3.7rem;`

Screenshot 2024-09-10 at 3 59 35 PM

@developer43  

1. Just a reminder that, since qualtrics doesn't render the form on my local I won't be able to validate with a screen capture that the form is rendering as expected - once on dev we can poke around and validate again.
2. this may be by design, but he height assisgnment to `.breadcrumb-spacing` and `#block-usagov-breadcrumbs` is different by `.3rem`

**Review by akf — APPROVED** [review]

**Pull request merged**